app/assets/javascripts/redactor-rails/config.js

```diff
@@ -0,0 +1,28 @@
+window.init_redactor = function(){
+  var csrf_token = $('meta[name=csrf-token]').attr('content');
+  var csrf_param = $('meta[name=csrf-param]').attr('content');
+  var params;
+  if (csrf_param !== undefined && csrf_token !== undefined) {
+      params = csrf_param + "=" + encodeURIComponent(csrf_token);
+  }
+  $('.redactor').redactor({
+        lang: 'pt_br',
+        buttons: ['formatting', 'bold', 'italic',
+'unorderedlist', 'orderedlist', 'outdent', 'indent',
+'image', 'link', 'alignment', 'horizontalrule'],
+        maxHeight: 800,
+        minHeight: 300,
+        convertVideoLinks: true,
+        convertUrlLinks: true,
+        // You can specify, which ones plugins you need.
+        // If you want to use plugins, you have add plugins to your
+        // application.js and application.css files and uncomment the line below:
+        // "plugins": ['fontsize', 'fontcolor', 'fontfamily', 'fullscreen', 'textdirection', 'clips'],
+        "imageUpload":"/redactor_rails/pictures?" + params,
+        "imageGetJson":"/redactor_rails/pictures",
+        "path":"/assets/redactor-rails",
+        "css":"style.css"
+      });
+}
+
+$(document).on( 'ready page:load', window.init_redactor );
```

Comment:
The problem is lib is included in application.js which is included for all catarse pages.
We want a separate file to include redactor only when needed.
I guess we could create a directory called vendor, but it might be confusing to have both vendor and lib.
Maybe move everything to a folder called optional ?